html2docx/html2docx.py

```diff
@@ -18,6 +18,20 @@
     "right": WD_ALIGN_PARAGRAPH.RIGHT,
     "justify": WD_ALIGN_PARAGRAPH.JUSTIFY,
 }
+BLOCK_ELEMENTS = (
```

Comment:
This can be set as it is only used with the `in` operator.

```suggestion
BLOCK_ELEMENTS = {
```